assets/ui/StructureTemplatePropertiesScreen.ui

```diff
@@ -65,6 +65,48 @@
               },
               "id": "typeRow"
             },
+                        {
```

Comment:
Looks like your indentation went off here from the existing style. Whether to use 2 spaces or 4 spaces in JSON is inconsistent across the codebase, here it was 2 before. Either is fine just best to keep the same within the entire file :-)

I reformatted the whitespace here after merging just to make it consistent within the file